coding-standards.md

```diff
@@ -236,6 +236,38 @@ to assess whether it's so urgent that you need to merge it
 regardless.  If you're not clear about this do ask about it by
 email or IRC.
 
+## Improving topic branches by rewriting history
+
+Generally, when you make a pull request based on a topic branch,
+the history of that branch should reflect how you ideally would
+have gone about making that change, rather than reflecting how
+you *actually* got to the end result. You're encouraged to use
+interactive rebase do this before asking anyone to look at a
```

Comment:
Your friendly drive-by proofreader notes a missing "to" in this line :)

Reply:
Oops, thanks - fixed now